EIPS/eip-future_salaries.md

```diff
@@ -0,0 +1,71 @@
+
+## Specification
+1. Finish the smart contract audit (it was already paid for) of [SalaryWithDAO](https://github.com/vporton/future-contracts/blob/master/contracts/SalaryWithDAO.sol) and [DefaultDAOInterface](https://github.com/vporton/future-contracts/blob/master/contracts/DefaultDAOInterface.sol) contracts.
+2. After the audit is finished, deploy these contracts (together with a DAOstack DAO controlling the `DefaultDAOInterface` with voting tokens initially allocated by a community consensus) and create an oracle ID using the `SalaryWithDAO` API.
+3. Audit and deploy [DonateETH](https://github.com/vporton/donations/blob/main/contracts/DonateETH.sol) redirecting ETH payments to this oracle, too.
+4. Discuss the fractions of miners fees and transfer fees of ETH to be transferred to `DonateETH` at every mining and transfer operation.
+5. Change the Ethereum clients to do these ETH transfers.
+
+In the future (it should be a separate EIP) we can switch to some more sophisticated DeFi scheme (`SalaryWithDAO` contract does support donations in DeFi token) of converting ETH to ERC-1155 tokens to be transferred to `SalaryWithDAO` than simply wrapping ETH into ERC-1155 token by `DonateETH`, to increase future value of the tokens transferred to `SalaryWithDAO`.
+
+Additional proposal for future consideration:
+
+- Consider (by the DAO voting) "obliging" the above mentioned DAO by a smart contract (with ownership transfer) to set the minimum oracle finish date in the future to some fixed value (need to discuss the exact number of years to lock funds for).
+- Consider (in a separate EIP) creating new oracles with shifted to father future (e.g. by one year forward once per year) the minimum finish times and spreading the transferred funds between several such oracles.
+
+## Rationale
+The Future Salaries is the _only_ known system of distributing significant funds to common good producers. (Quadratic funding aimed to do a similar thing, but in practice as we see on GitCoin it favors a few developers, ignores project of highly advanced scientific research that is hard to explain to an average developer, and encourages colluding, and it just highly random due to small number of donors. Also quadratic funding simply does not gather enough funds to cover common good needs). So this EIP is the only known way to recover the economy.
+
+The economical model of Future Salaries is described in [this research article preprint](https://github.com/vporton/gitcoin-web/blob/future/app/assets/docs/science-salaries.pdf).
+
+Funding multiple oracles with different finish time would alleviate the future trouble that the ETH (or other tokens) supply would suddenly increase when the oracle finishes.
+
+## Backwards Compatibility
+This EIP requires a hard fork, so we need some time for all clients to be updated. The change in clients is however trivial and the update won't require much time.
```

Comment:
Hard fork is assumed and it doesn't need to be mentioned.  You can just put
```suggestion
There are no known backward compatibility issues with this EIP.
```
You may want to think about if there are any possibilities with contracts or applications that have made assumptions about ETH transfers all occurring either as miner payments or transactions, as this introduces a new mechanism for moving ETH around the system.

Reply:
It can be implemented as ETH transactions. So, there is no such issue.

What to say in the EIP?

Reply:
Sorry, a wrong comment. Ignore it. I've edited the repository.